plpfuse/fuse.c

```diff
@@ -473,10 +486,15 @@ static int plp_setxattr(const char *path, const char *name, const char *value, s
     debuglog("setxattr succeeded");
     return 0;
   } else {
+    #ifdef __FreeBSD__
```

Comment:
Please use `#ifdef XATTR_REPLACE` (& similarly for `XATTR_CREATE`) rather than `#ifdef __FreeBSD__`. More general, and it's a safe bet that `XATTR_{REPLACE,CREATE} are preprocessor defines.

Reply:
I have now implemented this